src/routes/clinical/clinical-api.ts

```diff
@@ -172,6 +160,63 @@ const createClincalApiRouter = async (esClient: Client, egoClient: EgoClient) =>
 			});
 		}
 	});
+
+	router.use('/donors/data-for-all-files', async (req: AuthenticatedRequest, res) => {
```

Comment:
if a filter is passed, this should be read as "'some' data about 'some' of the files".

I think "all" here is either redundant or misleading, unless it actually means "'some' data for all the files", which is not immediately explicit in the name